packages/cfd/src/Containers/ctrader-derivx-trade-modal.tsx

```diff
@@ -397,16 +398,16 @@ const CTraderDerivXTradeModal = ({
                 <div className='cfd-trade-modal__download-center-options--mobile-links'>
                     <div className='cfd-trade-modal__download-center-options--mobile-links--apple'>
                         <a href={mobileDownloadLink(platform, 'ios')} target='_blank' rel='noopener noreferrer'>
-                            <Icon icon='IcInstallationApple' width={isMobile() ? '160' : '130'} height={40} />
+                            <StandaloneAppStoreIcon width={isMobile() ? '160' : '130'} height={40} />
                         </a>
                     </div>
                     <a href={mobileDownloadLink(platform, 'android')} target='_blank' rel='noopener noreferrer'>
-                        <Icon icon='IcInstallationGoogle' width={135} height={40} />
+                        <StandaloneAndroidIcon width={135} height={40} />
                     </a>
 
                     {platform !== CFD_PLATFORMS.CTRADER && (
                         <a href={mobileDownloadLink(platform, 'huawei')} target='_blank' rel='noopener noreferrer'>
-                            <Icon icon='IcInstallationHuawei' width={135} height={40} />
+                            <StandaloneHuaweiAppGalleryIcon width={135} height={40} />
```

Comment:
The same icons are being used in `getMobileDownloadOptions` just wondering if we can put this inside `constants` file so we can limit the amount of icons being imported.